waiter/test/waiter/scheduler_test.clj

```diff
@@ -53,6 +53,10 @@
     (locking lock
       (f))))
 
+(defn- extract-reference-type->reference-name-fn
```

Comment:
Nitpick: we don't usually give `defn` symbols names ending in `-fn`.

Reply:
Renamed.